xml/System.Text.Json/JsonTokenType.xml

```diff
@@ -94,7 +94,7 @@
       </ReturnValue>
       <MemberValue>9</MemberValue>
       <Docs>
-        <summary>The token type is the JSON literal "false".</summary>
+        <summary>The token type is the JSON literal <b>false</b>.</summary>
```

Comment:
We should fix "null" and "true" as well to follow this change to "false" (`TokenType.True`, `TokenType.Null`).

Reply:
@ahsonkhan can you please elaborate? What would you like me to do here?